spec/menu_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { described_class.new('menu.yaml') }
+  let(:dishes) { {:onion_bhaji=>2.95, :chicken_tikka_masala=>7.95, :naan=>1.95} }
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)